src/d3d8/d3d8_format.h

```diff
@@ -15,6 +15,17 @@ namespace dxvk {
     return isDXT(D3DFORMAT(fmt));
   }
 
+  inline constexpr bool isSupportedDepthStencilFormat(D3DFORMAT fmt) {
+    // native d3d8 doesn't support D3DFMT_D32, D3DFMT_D15S1 or D3DFMT_D24X4S4
```

Comment:
How do we know this? Is this just modern versions of d3d8 or did it always never support these? Does that also apply for all hardware?

Reply:
We've checked on native Nvidia and Intel and I've also written some test for these bits (see [d3d8-test](https://github.com/WinterSnowfall/d8vk-tests)). Both Nvidia/Intel report these formats as unsupported on Windows and WineD3D also does the same. This is why the games worked with WineD3D. As to what is happening behind the scenes (basically what is crashing) I am unsure. The games will crash if allowed to pick these formats, that much is certain.